drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua

```diff
@@ -32,6 +32,9 @@ local ENDPOINTS = {
   child = 2
 }
 
+-- This value is carried over from the DTH
+local ON_OFF_DELAY_SEC = 5
```

Comment:
you could use `zwave.constants.MIN_DIMMING_GET_STATUS_DELAY`

Reply:
Done